CSharp/Samples/Microsoft.Bot.Builder.Abstractions.Teams/TeamsInvokeActivityHandlerBase.cs

```diff
@@ -0,0 +1,125 @@
+﻿// <copyright file="TeamsInvokeActivityHandlerBase.cs" company="Microsoft">
+// Licensed under the MIT License.
+// </copyright>
+
+namespace Microsoft.Bot.Builder.Abstractions.Teams
+{
+    using System.Threading.Tasks;
+    using Microsoft.Bot.Schema.Teams;
+
+    /// <summary>
```

Comment:
Would this class even work in most cases? If the invoke responses were sent as null wouldn't BotFramework throw BadRequest?

Reply:
I copied the implementation of `Task.FromResult<InvokeResponse>(null);` from your Wiki example. Of course it won't work for each case since some cases it requires invoke response in a specific type. My intention is to have a default implementation so to prevent from breaking existing classes given adding more invokes in the future. Note that we're having a bunch of invoke handlers and we're adding more for sure. Having one more newly added interface will potential break all existing derived classes as they're directly implementing the set of interface and there's no choice to have optional interface implementation by default. To keep a default base class can avoid this design problem. An alternative solution is to throw exceptions for non-implemented methods instead of return null. Thought?